README.md

```diff
@@ -306,7 +306,7 @@ And, an equivalent configuration for Google Cloud Storage:
           parallel: 4
     }
 
-Note that GCS assumes the presence of a service account file and an environment variable of GOOGLE_APPLICATION_CREDENTIALS set pointing to this file. For example:
+Note that GCS assumes the presence of a service account file and an environment variable of `GOOGLE_APPLICATION_CREDENTIALS` set pointing to this file. For example:
```

Comment:
Could we upload the test script to `"test": "npm run testAzure && GOOGLE_APPLICATION_CREDENTIALS=gcs-credentials-uploadfstest.json mocha test/ && node test-imagemagick.js && eslint .",` with instruction that it's necessary for testing?